apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+
+### `processor_status`
+
+This table shares how current this processor's data is.
+
+gives you latest version processed per “processor”
+
+| Field                      | Type   | Description                                                                              |
+| -------------------------- | ------ | ---------------------------------------------------------------------------------------- |
+| last_success_version       | bigint | The version number of the last successful processor run. Ex. 5000000                     |
+| last_transaction_timestamp | String | Timestamp of the last processed transaction. Ex. "2024-04-17T02:14:25.68771"             |
+| last_updated               | String | Timestamp of the last update to this processor's status. Ex. "2024-04-17T02:14:25.68771" |
+| processor                  | String | Name of the processor. Ex. "TransactionProcessor"                                        |
+
+## NFT
+
+### `token_activities_v2`
+
+_Has an aggregate view for summary data called `token_activities_v2_aggregate`_
+
+This table tracks token activities and is especially useful for tracking NFT activity. This includes both v1 and v2 data.
+
+| Field                      | Type    | Description                                                                                                                              |
+| -------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| after_value                | String  | The value of a token property after the transaction. Ex. "100"                                                                           |
+| aptos_names_from           | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_from_aggregate | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_to             | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_to_aggregate   | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| before_value               | String  | The value of a token property before the transaction. Ex. "50"                                                                           |
+| current_token_data         | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| entry_function_id_str      | String  | The identifier of the function called in this transaction. Ex. "transfer"                                                                |
```

Comment:
This is actually the fully qualified path, so 0x1::aptos_account::transfer for example.